src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,32 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
```

Comment:
Ensure that the `UserInfo` component exists in the `../UserInfo` file and is correctly exported. If it doesn't exist, you need to create it or adjust the import path.